app/services/booking_bug_connection.rb

```diff
@@ -7,23 +7,21 @@ def initialize(config:)
   end
 
   def auth_token
+    Rails.logger.info('BookingBugConnection') { 'Retrieving authorisation token' }
```

Comment:
Faraday [has a logger](https://github.com/lostisland/faraday/blob/master/lib/faraday/response/logger.rb) and also [supports instrumentation](https://github.com/lostisland/faraday_middleware/wiki/Instrumentation). That might be a more preferable way of giving us visibility on latent parts of the application?